ddtrace/internal/ci_visibility/api/_test.py

```diff
@@ -54,28 +59,45 @@ def __init__(
         self._codeowners = codeowners
         self._source_file_info = source_file_info
         self._original_test: Optional[TestVisibilityTest] = None
-        self._is_early_flake_retry = is_early_flake_retry  # NOTE: currently unused
         self._exc_info: Optional[TestExcInfo] = None
         self._coverage_data: TestVisibilityCoverageData = TestVisibilityCoverageData()
 
         if self._parameters is not None:
             self.set_tag(test.PARAMETERS, parameters)
 
-        # Currently unsupported
+        self._is_new = is_new
+
+        self._is_efd_retry = is_efd_retry
+        self._efd_retries: List[TestVisibilityTest] = []
+        self._efd_abort_reason: Optional[str] = None
+        self._efd_initial_finish_time_ns: Optional[int] = None
+
+        # Currently unsupported]
```

Comment:
Oh, I missed this one the first time :P 
```suggestion
        # Currently unsupported
```